client/sections.js

```diff
@@ -835,10 +835,10 @@ const sections = [
 	{
 		name: 'a8c-for-agencies-client',
 		paths: [
-			'/agency/[a-zA-Z0-9]+/client/[a-zA-Z0-9]+$',
 			'/client/landing',
 			'/client/subscriptions',
 			'/client/payment-methods',
+			'/purchases/payment-methods/add',
```

Comment:
I'm not sure if this should be here.

Reply:
Good catch! Addressed it here: https://github.com/Automattic/wp-calypso/pull/91640/commits/ac029ed4001b56222fb68bda285d828e389a00b6